src/ituneslibrary.cs

```diff
@@ -327,6 +328,11 @@ interface ITLibPlaylist
 
 		[Export ("kind", ArgumentSemantic.Assign)]
 		ITLibPlaylistKind Kind { get; }
+
+		[Introduced (PlatformName.MacOSX, 10, 6, message: "Use 'isPrimary' instead.")]
+		[Deprecated (PlatformName.MacOSX, 12, 0, message: "Use 'isPrimary' instead.")]
+		[Export ("master")]
+		bool Master { [Bind ("isMaster")] get; }
```

Comment:
I see that you've moved the `Master` declaration here from above - but don't move code around unless really necessary, because it shows up as "removed+added" in the diff, which makes the diff harder to review.

On a more general note, changes that don't really change anything also makes it harder to run "git blame" to figure out when/what something happened, because the first thing that will come up is the commit that moved the code, which is not what you're looking for.

Reply:
To add to that, if we where working with a xcode13 branch, it would make my life harder when doing merges and back ports :/

Reply:
understood! fixed :)